_posts/news/2020-10-16-announcing-scalajs-1.3.0.md

```diff
@@ -0,0 +1,321 @@
+
+Bug reports can be filed [on GitHub](https://github.com/scala-js/scala-js/issues).
+
+## Release notes
+
+If upgrading from Scala.js 0.6.x, make sure to read [the release notes of Scala.js 1.0.0]({{ BASE_PATH }}/news/2020/02/25/announcing-scalajs-1.0.0/) first, as they contain a host of important information, including breaking changes.
+
+This is a **minor** release:
+
+* It is backward binary compatible with all earlier versions in the 1.x series: libraries compiled with 1.0.x through 1.2.x can be used with 1.3.0 without change.
+* It is *not* forward binary compatible with 1.2.x: libraries compiled with 1.3.0 cannot be used with 1.2.x or earlier.
+* It is *not* entirely backward source compatible: it is not guaranteed that a codebase will compile *as is* when upgrading from 1.2.x (in particular in the presence of `-Xfatal-warnings`).
+
+As a reminder, libraries compiled with 0.6.x cannot be used with Scala.js 1.x; they must be republished with 1.x first.
+
+## Module splitting
+
+### Quickstart
+
+Instead of `fastOptJS` / `fullOptJS`, use `fastLinkJS` / `fullLinkJS` and do any or all of the following:
+
+#### For many small modules
+
+{% highlight scala %}
+import org.scalajs.linker.interface.ModuleSplitStyle
+scalaJSLinkerConfig ~= (_.withModuleSplitStyle(ModuleSplitStyle.SmallestModules))
+{% endhighlight %}
+
+#### For different entry points
+
+Set the `moduleID` for your top level exports and/or module initializers explicitly.
+The default `moduleID` is `"main"`.
+
+{% highlight scala %}
+@JSExportTopLevel(name = "startAdmin", moduleID = "admin")
+def startAdmin(): Unit = ???
+{% endhighlight %}
+
+{% highlight scala %}
+import org.scalajs.linker.interface.ModuleInitializer
+scalaJSModuleInitializers in Compile += {
+  ModuleInitializer.mainMethod("my.app.admin.Main", "main")
+    .withModuleID("admin")
+}
+{% endhighlight %}
+
+Everything with the same `moduleID` will go into the same entry point module.
+
+### What is Module Splitting?
+
+With module splitting, the Scala.js linker splits its output into multiple JavaScript modules (i.e. files).
+Both ES6 modules (`ModuleKind.ESModule`) and CommonJS modules (`ModuleKind.CommonJSModule`) are supported.
+
+There are several reasons to split JavaScript output into multiple files:
+
+* Share code between different parts of an application (e.g. user/admin interfaces).
+* Create smaller files to minimize changes for incremental downstream tooling.
+* Load parts of a large app progressively (not supported yet, see [#4201](https://github.com/scala-js/scala-js/issues/4201)).
+  
+The Scala.js linker can split a full Scala.js application automatically based on:
+
+* The entry points (top level exports and module initializers)
+* The split style (fewest modules or smallest modules)
+
+### Entry Points
+
+Scala.js generated code has two different kinds of entry points:
```

Comment:
I believe this should be spelled "Scala.js-generated code", with a dash, shouldn't it?